src/renderer/dx/CustomTextRenderer.cpp

```diff
@@ -509,7 +509,7 @@ CATCH_RETURN()
     clipRect.top = origin.y + drawingContext->topClipOffset;
     clipRect.bottom = origin.y + drawingContext->cellSize.height - drawingContext->bottomClipOffset;
     clipRect.left = 0;
-    clipRect.right = drawingContext->targetSize.width;
+    clipRect.right = FLT_MAX;

```

Comment:
👀 

This seems... interesting. Any chance this breaks something by not clipping on the right side anymore? My attempted fix was (I believe) 

```c++
clipRect.left = origin.x;
```

on the line above, but I don't know which is better

Reply:
Hmm... I think that would work as well, but only if you also do `clipRect.right = origin.x + width`.
I believe `FLT_MAX` is correct, because we never limited the `left` border of the clip-rect either. 
This implies to me that the `clipRect.right = drawingContext->targetSize.width;` is just a hold-over from when we didn't draw entire rows at a time.